### PR TITLE
Fix typo in IP reporting for Hyper-V provider

### PR DESCRIPTION
**Pull request by caseygbigelow**
Fix for #12456 - I have not yet run tests because the scope is so small but will endeavour to do so if that's maintainers' preference. Cheers, folks.

Previously we were iterating over a list of `networks`, doing operations on each `network`. Now, we do operations on only one `netdev` (see 1148658a53b0ce15696e7e1b3df8042599950529), but the variable name was not changed. Correcting here.

### CODE REVIEW

**Comment by joelst** [comment]
@caseygbigelow Thank you for doing submitting this PR. I just spent a bit of time locating the issue and noticed that you already fixed it.

Are you able to sign the CLA so there are no blockers once it has been reviewed?

Right now Hyper-V deployments are broken until this is merged. It was an easy mistake to miss when the last PR was accepted. This should be an easy review.

**Review by joelst — APPROVED** [review]
This is correct. 

The last PR changed the name to $netdev above but did not carry that change down to this line. 

This will fix the problem.

**Comment by caseygbigelow** [comment]
image

CLA shows as signed in my browser. Clicking the "recheck" option in [the comment above](https://github.com/hashicorp/vagrant/pull/12457#issuecomment-879181745) does not affect any change. I see that the `license/cla` workflow shows as "Pending", with a note that "First-time contributors need a maintainer to approve running workflows"; maybe a maintainer needs to approve the workflow to recheck? I am not sure how my sign is supposed to propagate out to this PR.

@joelst, since you seem to have arrived at this issue independently, in the worst case would you be able to resubmit in the event that I can't get the CLA working and have to decline here? Also want to sheepishly note I haven't been able to get my env set up for Ruby development and run the tests...I know it's just PowerShell here but feel bad for not doing the due dilligence.

**Comment by joelst** [comment]
> image
> 
> CLA shows as signed in my browser. Clicking the "recheck" option in [the comment above](https://github.com/hashicorp/vagrant/pull/12457#issuecomment-879181745) does not affect any change. I see that the `license/cla` workflow shows as "Pending", with a note that "First-time contributors need a maintainer to approve running workflows"; maybe a maintainer needs to approve the workflow to recheck? I am not sure how my sign is supposed to propagate out to this PR.
> 
> @joelst, since you seem to have arrived at this issue independently, in the worst case would you be able to resubmit in the event that I can't get the CLA working and have to decline here? Also want to sheepishly note I haven't been able to get my env set up for Ruby development and run the tests...I know it's just PowerShell here but feel bad for not doing the due diligence.

As you mentioned it is an easy and straightforward fix so not much to test. I did test the change yesterday and it works fine. 

I can also do a PR if needed, but want to make sure this is a clear as possible for the reviewers to approve.

**Comment by caseygbigelow** [comment]
Fixed by #12472, thanks for taking care of this one @chrisroberts. Declining.

**Pull request closed without merging**
